src/plugins/discover/public/application/main/components/layout/discover_histogram_layout.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { RefObject } from 'react';
+import { UnifiedHistogramLayout } from '@kbn/unified-histogram-plugin/public';
+import { useDiscoverServices } from '../../../../hooks/use_discover_services';
+import { useDiscoverHistogram } from './use_discover_histogram';
+import type { DiscoverSearchSessionManager } from '../../services/discover_search_session';
+import type { InspectorAdapters } from '../../hooks/use_inspector';
+import { type DiscoverMainContentProps, DiscoverMainContent } from './discover_main_content';
+import { ResetSearchButton } from './reset_search_button';
+
+export interface DiscoverHistogramLayoutProps extends DiscoverMainContentProps {
+  resetSavedSearch: () => void;
+  isTimeBased: boolean;
+  resizeRef: RefObject<HTMLDivElement>;
+  inspectorAdapters: InspectorAdapters;
+  searchSessionManager: DiscoverSearchSessionManager;
+}
+
+export const DiscoverHistogramLayout = ({
+  isPlainRecord,
+  dataView,
+  resetSavedSearch,
+  savedSearch,
+  savedSearchData$,
+  state,
+  stateContainer,
+  isTimeBased,
+  resizeRef,
+  inspectorAdapters,
+  searchSessionManager,
+  ...mainContentProps
+}: DiscoverHistogramLayoutProps) => {
+  const services = useDiscoverServices();
+
+  const commonProps = {
+    dataView,
+    isPlainRecord,
+    stateContainer,
+    savedSearch,
+    state,
+    savedSearchData$,
+  };
+
+  const histogramProps = useDiscoverHistogram({
+    isTimeBased,
+    inspectorAdapters,
+    searchSessionManager,
+    ...commonProps,
+  });
+
+  if (!histogramProps) {
+    return null;
+  }
+
+  return (
+    <UnifiedHistogramLayout
+      resizeRef={resizeRef}
+      className="dscPageContent__inner"
```

Comment:
I'm not sure I understand this comment. This was migrated from `discover_main_content.tsx` which also used this class on `UnifiedHistogramLayout`.

If you're suggesting we get rid of the `dscPageContent__inner` class (and styles in `discover_layout.scss`), and instead use some local styles here, I think that would be fine since it doesn't seem like `dscPageContent__inner` is used anywhere else.